hebiros_advanced_examples/launch/omnibase/rosie_gmapping.launch

```diff
@@ -0,0 +1,77 @@
+<launch>
```

Comment:
I would go ahead and hold off on adding this file until we get all the dependent files in and cleaned up (e.g., the .xacro) so we can have this as a documented + functioning demo